firmware_revamp/can_api/megayaml_generator.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd 
+from pprint import pprint
+# import yaml
+from collections import OrderedDict 
+import oyaml as yaml
+import re
+
+
+data = pd.read_csv("address_space.csv") 
+
+# print(data.head())
+# lets start by just extracting the info thats therews
+output = {}
+for index, row in data.iterrows():
+    if index > 16 or index < 11:
+        continue
+    row_out = {}
+    row_out["can_id"] = row["CAN ID"].strip()
+    row_out["dec"] = row["(dec)"]
+
+    row_out['freq'] = row['Frequency (Hz)']
+    row_out['len'] = row['Message Length']
+    row_out['purpose'] = row['Purpose'].strip()
+
+    signal_names = []
+    for i in range(8):

```

Comment:
are there always 8 signals? can we make this a flexible range instead of hard-coded?